draft-yasskin-http-origin-signed-responses.md

```diff
@@ -147,24 +147,33 @@ to include in the `Signed-Headers` header field.
 
 The `Signature` header field conveys a list of signatures for an exchange, each
 one accompanied by information about how to determine the authority of and
-refresh that signature.
+refresh that signature. Each signature directly signs the significant headers of
+the exchange and identifies one of those headers that enforces the integrity of
+the exchange's payload.
 
 The `Signature` header is a Structured Header as defined by
 {{!I-D.ietf-httpbis-header-structure}}. Its value MUST be a list (Section 4.8 of
 {{!I-D.ietf-httpbis-header-structure}}) of parameterised labels (Section 4.4 of
 {{!I-D.ietf-httpbis-header-structure}}).
 
-Each parameterised label MUST have parameters named "sig", "validityUrl",
-"date", and "expires", and either "certUrl" and "certSha256" parameters or an
-"ed25519Key" parameter. This specification gives no meaning to the label itself,
-which can be used as a human-readable identifier for the signature (see {{parameterised-binary}}). The present
-parameters MUST have the following values:
+Each parameterised label MUST have parameters named "sig", "integrity",
+"validityUrl", "date", and "expires", and either "certUrl" and "certSha256"
```

Comment:
Not for this commit, but this line was a bit hard for me to understand. Can we split to something like:
```
Each parameterised label MUST have parameters named "sig", "integrity", "validityUrl", "date", and "expires".
In addition, each parameterised label MUST have either "certUrl" and "certSha256" parameters, or an "ed25519Key" parameter.
```

Reply:
Done.